packages/analytics-core/src/types/event/base-event.ts

```diff
@@ -37,6 +37,8 @@ export interface EventOptions {
   price?: number;
   quantity?: number;
   revenue?: number;
+  receipt?: string;
+  receiptSig?: string;
```

Comment:
Notes: 
the event.event_properties.$revenue is not documented and only the SDK does it
internally, event.revenue gets moved to event.event_properties.$revenue